src/main/java/uk/gov/hmcts/reform/refunds/services/RefundsServiceImpl.java

```diff
@@ -117,9 +117,20 @@
 
     @Override
     public RefundResponse initiateRefund(RefundRequest refundRequest, MultiValueMap<String, String> headers) throws CheckDigitException {
-        //validateRefundRequest(refundRequest); //disabled this validation to allow partial refunds
+
+        String paymnetMethod = null;
+
+        if (refundRequest.getPaymentMethod() != null) {
+
+            if (refundRequest.getPaymentMethod().equals("cheque") || refundRequest.getPaymentMethod().equals("cash")
+                || refundRequest.getPaymentMethod().equals("postal order") && (refundRequest.getPaymentChannel().equals("bulk scan"))) {
```

Comment:
Pls remove hard-coding and use constants

Reply:
This code is updated in base branch - **_instruction_type_** , hence issue resolved automatically